projects/avodata/firstStage.py

```diff
@@ -72,8 +79,11 @@ def run_check(m):
 
 # The professions are - driver, lawyer, programmer
 # The characteristics are - social status, working with people, analytical thinking
-c_matrix = [[-3, -3, -2],
-            [-1, 2, 3],
-            [-3, -3, -2]]
+# c_matrix = [[-3, -3, -2],
+#             [-1, 2, 3],
+#             [-3, -3, -2]]
+
+c_matrix = create_c_matrix_from_csv('check.csv')
```

Comment:
How about you add a `check.csv` example so people know how to run the code?
Also, would be good to explain the structure of `check.csv`.

Reply:
What you mean in the first sentence?